### PR TITLE
zh-trans: add docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md

### PR DESCRIPTION
**Pull request by SataQiu**

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+<!--
+## Overview
+-->
+## 概述
+
+<!--
+A kubelet's HTTPS endpoint exposes APIs which give access to data of varying sensitivity,
+and allow you to perform operations with varying levels of power on the node and within containers.
+-->
+kubelet 的 HTTPS endpoint 端点公开了 API，可以访问不同敏感度的数据，
```
Comment (xichengliudui) [inline content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md]: endpoint 端点

File: content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+并允许您在节点和容器内执行不同权限级别的操作。
+
+<!--
+This document describes how to authenticate and authorize access to the kubelet's HTTPS endpoint.
+-->
+本文档介绍如何认证和授权访问 kubelet 的 HTTPS endpoint 端点。
```
Comment (xichengliudui) [inline content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md]: endpoint 端点

File: content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+## Kubelet authentication
+-->
+## Kubelet 认证
+
+<!--
+By default, requests to the kubelet's HTTPS endpoint that are not rejected by other configured
+authentication methods are treated as anonymous requests, and given a username of `system:anonymous`
+and a group of `system:unauthenticated`.
+-->
+默认情况下，未被其他已配置的身份验证方法拒绝的对 kubelet 的 HTTPS endpoint 端点的请求将被视为匿名请求，
```
Comment (xichengliudui) [inline content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md]: endpoint 端点

File: content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+<!--
+* start the kubelet with the `--anonymous-auth=false` flag
+-->
+* 附带 `--anonymous-auth=false` 参数启动 kubelet
+
+<!--
+To enable X509 client certificate authentication to the kubelet's HTTPS endpoint:
+-->
+要为 kubelet 的 HTTPS endpoint 端点启用 X509 客户端证书身份验证，请执行以下操作：
```
Comment (xichengliudui) [inline content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md]: endpoint 端点

Comment (SataQiu) [reply content/zh/docs/reference/command-line-tools-reference/kubelet-authentication-authorization.md]: Done

**Pull request merged**